app/components/Contacts/Network/Network.js

```diff
@@ -289,13 +244,11 @@ class Network extends Component {
                         <Box
                           mr={2}
                           className="hint--right"
-                          data-hint={intl.formatMessage({ ...messages[status] })}
+                          data-hint={intl.formatMessage({ ...messages[display_status] })}
                         >
-                          <StatusIndicator variant={status} />
+                          <StatusIndicator variant={display_status} />
                         </Box>
-                        <Text css={{ '&:hover': { opacity: 0.5 } }}>
-                          {displayNodeName(channel)}
-                        </Text>
+                        <Text css={{ '&:hover': { opacity: 0.5 } }}>{display_name}</Text>
```

Comment:
Would be nice to use styled here

Reply:
I want to try to limit the scope of this PR to resolving some of the search/filter related issues and don't want to get into refactoring out into separate components here. We have already done that in the `next` branch.